Upstreams  September

doob187 opened the pull request:
Add all here for upstream pushes for September upstream:

aelfa committed:
add cloudflare-ddns

drag0n141 committed:
Update cloudflare-ddns.yml

aelfa committed:
add `dockupdater` label

aelfa committed:
Add `dockupdater` label | Fix rutorrent yml.

aelfa committed:
Block Direct Connection

dependabot[bot] committed:
pip-update(deps): bump mkdocs-material from 7.2.5 to 7.2.6 in /wiki

Bumps [mkdocs-material](https://github.com/squidfunk/mkdocs-material) from 7.2.5 to 7.2.6.
- [Release notes](https://github.com/squidfunk/mkdocs-material/releases)
- [Changelog](https://github.com/squidfunk/mkdocs-material/blob/master/docs/changelog.md)
- [Commits](https://github.com/squidfunk/mkdocs-material/compare/7.2.5...7.2.6)

---
updated-dependencies:
- dependency-name: mkdocs-material
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

drag0n141 committed:
Add Uptime Kuma

drag0n141 committed:
Update uptime-kuma.yml

doob187 committed:
Update uptime-kuma.yml

Ending and top

dependabot[bot] committed:
pip-update(deps): bump mkdocs-git-revision-date-localized-plugin

Bumps [mkdocs-git-revision-date-localized-plugin](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin) from 0.9.2 to 0.9.3.
- [Release notes](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/releases)
- [Commits](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/compare/v0.9.2...v0.9.3)

---
updated-dependencies:
- dependency-name: mkdocs-git-revision-date-localized-plugin
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
pip-update(deps): bump mkdocs-material-extensions in /wiki

Bumps [mkdocs-material-extensions](https://github.com/facelessuser/mkdocs-material-extensions) from 1.0.1 to 1.0.2.
- [Release notes](https://github.com/facelessuser/mkdocs-material-extensions/releases)
- [Changelog](https://github.com/facelessuser/mkdocs-material-extensions/blob/master/changelog.md)
- [Commits](https://github.com/facelessuser/mkdocs-material-extensions/compare/1.0.1...1.0.2)

---
updated-dependencies:
- dependency-name: mkdocs-material-extensions
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

aelfa committed:
FAILSAFE: backups don't get overwritten.

aelfa committed:
This makes more sense

aelfa committed:
added description for `tag`

doob187 committed:
Update migration.md

Layout and reworks

aelfa committed:
repo cleanup!

dependabot[bot] committed:
pip-update(deps): bump mkdocs-material-extensions in /wiki

Bumps [mkdocs-material-extensions](https://github.com/facelessuser/mkdocs-material-extensions) from 1.0.2 to 1.0.3.
- [Release notes](https://github.com/facelessuser/mkdocs-material-extensions/releases)
- [Changelog](https://github.com/facelessuser/mkdocs-material-extensions/blob/master/changelog.md)
- [Commits](https://github.com/facelessuser/mkdocs-material-extensions/compare/1.0.2...1.0.3)

---
updated-dependencies:
- dependency-name: mkdocs-material-extensions
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

aelfa committed:
Add `krusader` to addons.

aelfa committed:
Add `plex-update-tool` && API's section.

aelfa committed:
update

aelfa committed:
Add `plex-meta-manager` to addons.

aelfa committed:
Added `plex-meta-manager` variables to environment

aelfa committed:
Added more to environment.

aelfa committed:
update

aelfa committed:
Added `environment variables` to traefik deploy.

aelfa committed:
update

aelfa committed:
Revert rookie mistake

aelfa committed:
Add `webtop`

doob187 committed:
Update webtop.yml

doob187 committed:
Update webtop.yml

aelfa committed:
add to environment

aelfa committed:
add to environment

aelfa committed:
Merge branch 'dev' of github.com:aelfa/dockserver into dev

drag0n141 committed:
Add Homer and Dashy

drag0n141 committed:
Add Homer and Dashy

drag0n141 committed:
Add Organizr

drag0n141 committed:
Add Organizr

aelfa commented:
@DrAg0n141 Organizr already added under:
apps/selfhosted/organizr.yml

drag0n141 committed:
Delete organizr.yml

drag0n141 committed:
Delete organizr.yml

drag0n141 commented:
> @DrAg0n141 Organizr already added under:
> apps/selfhosted/organizr.yml

Thanks delete :)

aelfa commented on apps/addons/dashy.yml:
```diff
@@ -0,0 +1,34 @@
+    security_opt:
+      - 'no-new-privileges:true'
+    volumes:
+      - '${APPFOLDER}/dashy/conf.yml:/app/public/conf.yml:rw'
```
@DrAg0n141 Please fix this!

doob187 replied:
Thiw will be super hard to code 
It's so massively customizable parts 

So I thinks is mus show the link  and  the after press enter just create a empty file and open them with  

```sh
sudo nano ${APPFOLDER}/dashy/conf.yml
```

All other should be made to much problems and you lose the custome parts

aelfa reviewed (commented):
Needs Sub-action.

aelfa committed:
Merge pull request #210 from aelfa/dev

doob187 committed:
Create dashy.yml

doob187 committed:
Create dashy.yml

doob187 committed:
Create dashy.sh

doob187 committed:
Create dashy.sh

doob187 committed:
Update dashy.sh

doob187 committed:
Update dashy.sh

drag0n141 committed:
Update ubuntu.sh

drag0n141 committed:
Update ubuntu.sh

doob187 committed:
Create ubuntu.sh

doob187 committed:
Create ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update dashy.yml

doob187 committed:
Update dashy.yml

doob187 committed:
Update dashy.yml

doob187 committed:
Update dashy.yml

drag0n141 committed:
Update dashy.yml

drag0n141 committed:
Update dashy.yml

doob187 committed:
Update dashy.yml

doob187 committed:
Update dashy.yml

doob187 committed:
Rename dashy.yml to dashy.j2

doob187 committed:
Rename dashy.yml to dashy.j2

doob187 committed:
Update dashy.sh

doob187 committed:
Update dashy.sh

doob187 committed:
Create config.yml

doob187 committed:
Create config.yml

doob187 committed:
Rename config.yml to config.yaml

doob187 committed:
Rename config.yml to config.yaml

doob187 committed:
Create docker.yml

doob187 committed:
Create docker.yml

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update dockserver

doob187 committed:
Update dockserver

doob187 commented:
Added dockserver docker updater

dependabot[bot] committed:
pip-update(deps): bump mkdocs-git-revision-date-localized-plugin

Bumps [mkdocs-git-revision-date-localized-plugin](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin) from 0.9.3 to 0.10.0.
- [Release notes](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/releases)
- [Commits](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/compare/v0.9.3...v0.10.0)

---
updated-dependencies:
- dependency-name: mkdocs-git-revision-date-localized-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
pip-update(deps): bump mkdocs-git-revision-date-localized-plugin

Bumps [mkdocs-git-revision-date-localized-plugin](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin) from 0.9.3 to 0.10.0.
- [Release notes](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/releases)
- [Commits](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/compare/v0.9.3...v0.10.0)

---
updated-dependencies:
- dependency-name: mkdocs-git-revision-date-localized-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

doob187 committed:
Merge pull request #211 from dockserver/dependabot/pip/wiki/dev/mkdocs-git-revision-date-localized-plugin-0.10.0

pip-update(deps): bump mkdocs-git-revision-date-localized-plugin from 0.9.3 to 0.10.0 in /wiki

doob187 committed:
Update dashy.sh

doob187 committed:
Update dashy.sh

doob187 reviewed (commented):
Add fail-safe to change the owner permission

aelfa committed:
Added `cloudflared`

aelfa committed:
Added `cloudflared`

aelfa committed:
Merge branch 'dockserver:dev' into dev

doob187 committed:
Update cloudflare-tunnel.yml

doob187 committed:
Update cloudflare-tunnel.yml

doob187 committed:
Merge pull request #212 from aelfa/dev

Added `cloudflared`

doob187 committed:
Create config.yaml

doob187 committed:
Create config.yaml

doob187 committed:
Update config.yaml

doob187 committed:
Update config.yaml

doob187 committed:
Update .env

doob187 committed:
Update .env

doob187 committed:
Update envmigrate.sh

doob187 committed:
Update envmigrate.sh

aelfa committed:
update

aelfa committed:
update

doob187 committed:
Merge pull request #213 from aelfa/dev

update

aelfa committed:
fix image tag for cloudflared.

aelfa committed:
fix image tag for cloudflared.

aelfa committed:
Merge branch 'dockserver:dev' into dev

aelfa committed:
Merge pull request #214 from aelfa/dev

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update docker-compose.yml

doob187 committed:
Update docker-compose.yml

aelfa committed:
Add `pihole-cloudflared` && `pihole-unbound`

aelfa committed:
Add `pihole-cloudflared` && `pihole-unbound`

aelfa committed:
Merge branch 'dockserver:dev' into dev

doob187 committed:
Merge pull request #215 from aelfa/dev

Add `pihole-cloudflared` && `pihole-unbound`

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update envmigrate.sh

doob187 committed:
Update envmigrate.sh

doob187 committed:
Update .env

doob187 committed:
Update .env

doob187 committed:
Update envmigrate.sh

doob187 committed:
Update envmigrate.sh

aelfa committed:
Add file check.

aelfa committed:
Add file check.

aelfa committed:
update

aelfa committed:
update

doob187 committed:
Update ubuntu.sh

Will fix 

https://github.com/dockserver/dockserver/issues/179

doob187 committed:
Update ubuntu.sh

Will fix 

https://github.com/dockserver/dockserver/issues/179

doob187 committed:
Merge pull request #216 from aelfa/dev

Add file check.

aelfa committed:
update

aelfa committed:
Merge branch 'dockserver:dev' into dev

aelfa committed:
update

aelfa committed:
Merge branch 'dev' of github.com:aelfa/dockserver into dev

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Merge branch 'dev' into dev

doob187 committed:
Merge pull request #217 from aelfa/dev

Added `temp` persmissions for the deploy

aelfa committed:
TEST

aelfa committed:
REVERT!!!

doob187 committed:
Update ubuntu.sh

aelfa committed:
Finally!!!!!!!!!!!!

drag0n141 committed:
Update .env

drag0n141 committed:
Update configuration.yml

drag0n141 committed:
Update .env

drag0n141 committed:
Update .env

drag0n141 committed:
Add Aria

drag0n141 committed:
Add Socket-Proxy

drag0n141 committed:
Update socket-proxy.yml

drag0n141 committed:
Update docker-compose.yml

drag0n141 committed:
Update socket-proxy.yml

aelfa committed:
Exposing traefik ports not needed with cloudflared

aelfa committed:
Static IPs

aelfa committed:
ADDED: Static IPs for `system` section

aelfa committed:
Added Static IPs for `mediamanager` section

aelfa committed:
ADDED: static IPs for `mediaserver` section.

aelfa committed:
redid static IPs for initial deploy.

aelfa committed:
static IPs for download clients section

aelfa commented:
> Included static IPs for multiple sections with the following details.


1. **mediamanager**:         `172.18.0.100-123`
2. **downloadclients**:      `172.18.0.124-137`
3. **coding**:               `172.18.0.138-139`
4. **system**:               `172.18.0.140-146`
5. **traefik+authelia etc**: `172.18.0.250-254`
6. **mediaserver**:          `172.18.0.220-223`

aelfa committed:
Added static IPs for coding section.

dependabot[bot] committed:
GH updates(deps): Bump ChloePlanet/github-tag-action from 2.0.0 to 2.0.3

Bumps [ChloePlanet/github-tag-action](https://github.com/ChloePlanet/github-tag-action) from 2.0.0 to 2.0.3.
- [Release notes](https://github.com/ChloePlanet/github-tag-action/releases)
- [Commits](https://github.com/ChloePlanet/github-tag-action/compare/2.0.0...2.0.3)

---
updated-dependencies:
- dependency-name: ChloePlanet/github-tag-action
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

aelfa committed:
Second redo for crucial services.

aelfa committed:
basic syntax fix

aelfa committed:
testing commit signature

drag0n141 committed:
Update socket-proxy.yml

drag0n141 committed:
Add Watchtower

doob187 committed:
Merge pull request #218 from dockserver/dependabot/github_actions/dev/ChloePlanet/github-tag-action-2.0.3

GH updates(deps): Bump ChloePlanet/github-tag-action from 2.0.0 to 2.0.3

doob187 commented on apps/system/watchtower.yml:
```diff
@@ -0,0 +1,32 @@
+      - 'PUID=${ID}
+      - 'PGID=${ID}'
+      - 'TZ=${TZ}'
+      - 'WATCHTOWER_CLEANUP=true'
```
Environment  !!!!

drag0n141 replied:
For Cleanup?

doob187 replied:
It's better for the hole part 

The goal is ..  all can be changed  <| the goal of DS |>

doob187 reviewed (commented):
See notes

doob187 committed:
Update pihole-cloudflared.yml

aelfa committed:
Add missing quote | Add more to `.env`

aelfa committed:
Add more to `.env`

doob187 reviewed (commented):
Env?!
```sh

- 'WATCHTOWER_REMOVE_VOLUMES=true' 
- 'WATCHTOWER_INCLUDE_STOPPED=false' 
- 'WATCHTOWER_NO_STARTUP_MESSAGE=true'@@ -18,7 +18,7 @@ services:

 - 'WATCHTOWER_ROLLING_RESTART=false'
 -  - 'WATCHTOWER_TIMEOUT=180s'

```

aelfa commented:
> 
> 
> Env?!
> 
> ```shell
> - 'WATCHTOWER_REMOVE_VOLUMES=true' 
> - 'WATCHTOWER_INCLUDE_STOPPED=false' 
> - 'WATCHTOWER_NO_STARTUP_MESSAGE=true'@@ -18,7 +18,7 @@ services:
> 
>  - 'WATCHTOWER_ROLLING_RESTART=false'
>  -  - 'WATCHTOWER_TIMEOUT=180s'
> ```

@doob187 [`acf58db` (#203)](https://github.com/dockserver/dockserver/pull/203/commits/acf58dbe3cde77af9e5aa7e85d6db16afb71af22)

aelfa committed:
update

aelfa committed:
minor typo fix

aelfa committed:
update

drag0n141 committed:
Update watchtower.yml

doob187 committed:
Update ubuntu.sh

Correcting  line syntax 
Add plex claim env fail safe

doob187 committed:
Update mount.yml

Remove docker.sock

doob187 committed:
Update uploader.yml

We don't need the ports for uploader

doob187 committed:
Update mount.yml

We don't need this ports here

aelfa commented:
@doob187 @DrAg0n141 The basic-proxy network is defined but not created.

doob187 commented:
https://github.com/dockserver/dockserver/blob/241f2f38854b9cae25d17077307d3c1d616ab234/preinstall/installer/ubuntu.sh#L106


Just proxy will created

aelfa committed:
update

aelfa commented:
> 
> 
> https://github.com/dockserver/dockserver/blob/241f2f38854b9cae25d17077307d3c1d616ab234/preinstall/installer/ubuntu.sh#L106
> 
> Just proxy will created

We need to create the `proxy-basic` network for socket-proxy.

drag0n141 committed:
Update netdata.yml

drag0n141 committed:
Update netdata.yml

drag0n141 committed:
Update ubuntu.sh

drag0n141 commented:
> > https://github.com/dockserver/dockserver/blob/241f2f38854b9cae25d17077307d3c1d616ab234/preinstall/installer/ubuntu.sh#L106
> > 
> > Just proxy will created
> 
> We need to create the `proxy-basic` network for socket-proxy.

Its now added.

drag0n141 committed:
Update .env

drag0n141 committed:
Update glances.yml

drag0n141 committed:
Update netdata.yml

drag0n141 committed:
Update dozzle.yml

drag0n141 committed:
Update diun.yml

drag0n141 committed:
Update heimdall.yml

drag0n141 committed:
Update statping.yml

drag0n141 committed:
Update cloud9.yml

drag0n141 committed:
Update webtop.yml

drag0n141 committed:
Update portainer.yml

drag0n141 committed:
Update dockupdater.yml

drag0n141 committed:
Update dockupdater.yml

drag0n141 committed:
Update netdata.yml

drag0n141 committed:
Socket-Proxy Change

drag0n141 committed:
Update ubuntu.sh

drag0n141 committed:
Update docker-compose.yml

doob187 committed:
Merge branch 'master' into dev

aelfa committed:
house cleanup

aelfa committed:
value `authrmapp2` unused without double quotes.

doob187 committed:
Merge branch 'master' into dev

drag0n141 committed:
Add Dashmachine

justinglock40 committed:
Update plex.yml

Makes video thumbnails smaller to keep /opt/appdata/plex from becoming overly bloated (Saves GB in size, currently using in my production server). Serves as a background function. Only works if the user enables "create video thumbnails on the library."  "Otherwise remains dormant.

dependabot[bot] committed:
pip-update(deps): bump mkdocs-material from 7.2.6 to 7.2.8 in /wiki

Bumps [mkdocs-material](https://github.com/squidfunk/mkdocs-material) from 7.2.6 to 7.2.8.
- [Release notes](https://github.com/squidfunk/mkdocs-material/releases)
- [Changelog](https://github.com/squidfunk/mkdocs-material/blob/master/docs/changelog.md)
- [Commits](https://github.com/squidfunk/mkdocs-material/compare/7.2.6...7.2.8)

---
updated-dependencies:
- dependency-name: mkdocs-material
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

doob187 committed:
Merge pull request #220 from dockserver/dependabot/pip/wiki/dev/mkdocs-material-7.2.8

pip-update(deps): bump mkdocs-material from 7.2.6 to 7.2.8 in /wiki

dependabot[bot] committed:
pip-update(deps): bump mkdocs-minify-plugin from 0.4.0 to 0.4.1 in /wiki

Bumps [mkdocs-minify-plugin](https://github.com/byrnereese/mkdocs-minify-plugin) from 0.4.0 to 0.4.1.
- [Release notes](https://github.com/byrnereese/mkdocs-minify-plugin/releases)
- [Commits](https://github.com/byrnereese/mkdocs-minify-plugin/compare/0.4.0...0.4.1)

---
updated-dependencies:
- dependency-name: mkdocs-minify-plugin
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

doob187 committed:
Merge pull request #222 from dockserver/dependabot/pip/wiki/dev/mkdocs-minify-plugin-0.4.1

pip-update(deps): bump mkdocs-minify-plugin from 0.4.0 to 0.4.1 in /wiki

drag0n141 committed:
Update netdata.yml

aelfa committed:
update conreq

aelfa committed:
Add Droplarr.

aelfa committed:
update

aelfa committed:
update

aelfa committed:
Add maximux

aelfa committed:
update port for maximux

dependabot[bot] committed:
pip-update(deps): bump mkdocs-material from 7.2.8 to 7.3.0 in /wiki

Bumps [mkdocs-material](https://github.com/squidfunk/mkdocs-material) from 7.2.8 to 7.3.0.
- [Release notes](https://github.com/squidfunk/mkdocs-material/releases)
- [Changelog](https://github.com/squidfunk/mkdocs-material/blob/master/docs/changelog.md)
- [Commits](https://github.com/squidfunk/mkdocs-material/compare/7.2.8...7.3.0)

---
updated-dependencies:
- dependency-name: mkdocs-material
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

doob187 committed:
Merge pull request #227 from dockserver/dependabot/pip/wiki/dev/mkdocs-material-7.3.0

pip-update(deps): bump mkdocs-material from 7.2.8 to 7.3.0 in /wiki

doob187 committed:
Update Plex.yml for Thumbnail 

Update plex.yml

dependabot[bot] committed:
GH updates(deps): Bump actions/github-script from 4 to 5

Bumps [actions/github-script](https://github.com/actions/github-script) from 4 to 5.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

doob187 committed:
Merge pull request #228 from dockserver/dependabot/github_actions/dev/actions/github-script-5

GH updates(deps): Bump actions/github-script from 4 to 5

aelfa committed:
update mount

aelfa committed:
bump cloudflared version

fscorrupt committed:
Update for Bypassing Uptime-Kuma Status Page

With this Bypass rule we can use authelia for Uptime-Kuma, but also share the Status Page to users without the need to Authenticate!

drag0n141 committed:
Merge pull request #229 from FSCorrupt/patch-1

Update for Bypassing Uptime-Kuma Status Page

dependabot[bot] committed:
pip-update(deps): bump pymdown-extensions from 8.2 to 9.0 in /wiki

Bumps [pymdown-extensions](https://github.com/facelessuser/pymdown-extensions) from 8.2 to 9.0.
- [Release notes](https://github.com/facelessuser/pymdown-extensions/releases)
- [Commits](https://github.com/facelessuser/pymdown-extensions/compare/8.2...9.0)

---
updated-dependencies:
- dependency-name: pymdown-extensions
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

doob187 committed:
Merge pull request #230 from dockserver/dependabot/pip/wiki/dev/pymdown-extensions-9.0

pip-update(deps): bump pymdown-extensions from 8.2 to 9.0 in /wiki

doob187 committed:
Update ubuntu.sh

Use v2

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 commented:
Each docker-compose  command 
Need to changed to  docker compose

Mass edit  will faster replace this

doob187 committed:
Update ubuntu.sh

drag0n141 committed:
Fix Docker Compose

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

drag0n141 committed:
Fix Command

doob187 committed:
Update ubuntu.sh

drag0n141 committed:
Fix Command

drag0n141 committed:
Merge branch 'dev' of https://github.com/dockserver/dockserver into dev

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Update ubuntu.sh

doob187 committed:
Merge branch 'master' into dev

doob187 committed:
Update ubuntu.sh

doob187 commented:
#major
